snuba/datasets/storages/__init__.py

```diff
@@ -1,44 +1,73 @@
-from enum import Enum
 from functools import lru_cache
+from glob import glob
+from typing import Any
 
+from yaml import safe_load
```

Comment:
I don't think we should directly load YAML outside of the `build_storage` code path

Reply:
I agree, but the `storage_builder` file imports the Enum from this file so it creates a circular import. Either all of the storage building code is moved into this init file or the `StorageKey` definition is moved into that file. I didn't think either way was very clean so this is what I was left with.

I'm open to any other suggestions you may have.

Reply:
Alright I've moved the loading out into `loader.py` since `built_storage` is in a file that imports the `StorageKey` enum which would cause a circular import